ui/src/components/Form/SelectBox/style.module.scss

```diff
@@ -86,6 +87,14 @@
   width: $size;
 }
 
+.llogo {
```

Comment:
Let's pick a better name for this class; if we are referring to `Logo` prop, we can name it `Logo` or `Logo-container` if provided Logo takes care of its own styles (as it should).